lib/gollum-lib/page.rb

```diff
@@ -281,6 +281,14 @@ def versions(options = {})
       @wiki.repo.git.versions_for_path(@path, @wiki.ref, log_pagination_options(options))
     end
 
+    # Public: The last version that has touched the Page. Can be nil.
+    #
+    # Returns Gollum::Git::Commit, or nil.
+    def last_version
+      return @last_version if defined? @last_version
```

Comment:
`defined?` will return the string `"instance-variable"` (which is truthy) even if `@last_version` is nil.
```
jruby-1.7.19 :001 > @last_version = nil
 => nil 
jruby-1.7.19 :002 > defined? @last_version
 => "instance-variable"
```
Is this the behavior you want? I would think `return @last_version if @last_version` is preferable, or not?